lib/install.ps1

```diff
@@ -34,6 +34,10 @@ function install_app($app, $architecture, $global, $suggested, $use_cache = $tru
         return
     }
 
+    if ((default_architecture -neq "arm64") -and ($architecture -eq "arm64")) {
```

Comment:
`(default_architecture) -ne "arm64")`  otherwise the "arm64" will be passed to the function as parameter.